### PR TITLE
merging sign and monotonicity columns into "DCP properties"

### PR DESCRIPTION
**Pull request by Transurgeon**
## Description
This PR is a follow-up to #2563 in hopes of addressing the suggestions in #2559. 
The changes are made for two tables in the "atomic functions" section of the docs.
@rileyjmurray @SteveDiamond what do you think of the combined DCP properties column?
I made some slight adjustments, adding "sign" and "monotonicity" when it was otherwise ambiguous.
Also for previously None instances, I just deleted those from the table. 
[Link](https://cvxpy--61.org.readthedocs.build/en/61/tutorial/functions/index.html#scalar-functions) for preview (everything fits without scrolling for the atoms table!). 

This PR also fixes the solver support table (see below).
image

## Type of change
- [ ] New feature (backwards compatible)
- [ ] New feature (breaking API changes)
- [ ] Bug fix
- [x] Other (Documentation, CI, ...)

## [Contribution checklist](https://www.cvxpy.org/contributing/index.html#contribution-checklist)
- [x] Add our license to new files.
- [x] Check that your code adheres to our coding style.
- [x] Write unittests.
- [x] Run the unittests and check that they’re passing.
- [x] Run the benchmarks to make sure your change doesn’t introduce a regression.

### CODE REVIEW

File: doc/source/tutorial/functions/index.rst [hunk 1]
```diff
@@ -213,7 +213,8 @@ and returns a scalar.
 
    * - :ref:`mixed_norm(X, p, q) <mixed-norm>`
 
-     - :math:`\left(\sum_k\left(\sum_l\lvert x_{k,l}\rvert^p\right)^{q/p}\right)^{1/q}`
+       :math:`\text{inner} = \left(\sum_l\lvert x_{k,l}\rvert^p\right)`
+     - :math:`\left(\sum_k \text{inner}^{q/p}\right)^{1/q}`
```
Comment (Transurgeon) [inline doc/source/tutorial/functions/index.rst:217]: image
This looks like this. Let me know what you think. The other option is to replace ``inner`` with ``Y=...`` similar to total_variation atoms below.

Comment (PTNobel) [reply doc/source/tutorial/functions/index.rst:217]: I prefer Y instead of inner

**Review by PTNobel — APPROVED** [review]
Looks good to me :)

**Comment by SteveDiamond** [comment]
Should the readthedocs be there?

**Comment by PTNobel** [comment]
> Should the readthedocs be there?

That needs to change before merge

**Comment by Transurgeon** [comment]
> > Should the readthedocs be there?
> 
> That needs to change before merge

done. thanks for the review!

**Review by SteveDiamond — APPROVED** [review]

**Pull request merged**